### PR TITLE
Add timestamps to TestReport objects

### PR DESCRIPTION
**Pull request by HTRafal**
Implements #19

### CODE REVIEW

File: tests/test_reportlog.py [hunk 1]
```diff
@@ -47,13 +47,16 @@ def test_fail():
 
     # rest of the json objects should be unserialized into report objects; we don't test
     # the actual report object extensively because it has been tested in ``test_reports``
-    # already.
+    # already (except start and stop attributes which are set by this plugin).
```
Comment (hterik) [inline tests/test_reportlog.py:50]: Edge-cases don't sound so good. Should it be upstreamed even higher into pytest itself?

Comment (HTRafal) [reply tests/test_reportlog.py:50]: That was my concern as well. pytest-reportlog responsiblity was only to dump test reports to a json file. It wasn't adding any new fields. A clean way to do this would be to either upstream to pytest itself (worst case create a new extension just for that?)

I'll try to prepare a new PR to pytest and let's see how it looks like

Comment (HTRafal) [reply tests/test_reportlog.py:50]: Worst case we can also add that hook locally to our setup
